### ISSUE TITLE
Is it possible to get the old behaviour?

### ISSUE THREAD

**Issue by da0ist** [WesBosch/brunch-toolkit#9]
I THINK my system is ready to update to 93, but after updating via the latest toolkit, I came up at 92. With the old version I could tell what was going on, but the new one just does stuff and doesn't tell you what it's doing.

**Comment by da0ist** [comment]
Yeah, I downloaded the old version and am updating to 93 no problem.

**Comment by WesBosch** [comment]
yeah, I really need to impliment a better error message handler. I'm not
sure what's causing this bad update issue though
